morphos/umbrel-app.yml

```diff
@@ -0,0 +1,27 @@
+manifestVersion: 1
+id: morphos
+category: files
+name: Morphos Server
+version: "0.1.1"
+tagline: Self-hosted file converter server
```

Comment:
Up to you here, but feel free to remove "server" from the tag so that the average user just understands the app as a self-hosted file converter